src/julia_internal.h

```diff
@@ -220,6 +220,7 @@ extern volatile size_t profile_bt_size_max;
 extern volatile size_t profile_bt_size_cur;
 extern volatile int profile_running;
 extern volatile int profile_all_tasks;
+extern int heartbeat_tid; // Mostly used to ensure we skip this thread in the CPU profiler. XXX: not implemented on Windows
```

Comment:
What does XXX mean? Is that a standard comment prefix in julia? I might have used NOTE or TODO for this

Reply:
Is just a prefix that people use to indicate "this code needs attention".

There are a few uses in the Julia codebase: e.g. [here](https://github.com/JuliaLang/julia/blob/0741f9b99587e90333b736999110fba3d1bc6cac/base/reshapedarray.jl#L363).

`TODO` would probably not be applicable here because I don't see us wanting to port this to Windows in the foreseeable future.